<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="1" ignored="8" total="11" passed="0">
  <reporter-output>
    <line>
      <![CDATA[<br><b>Please look to the screenshot - </b>]]>
    </line>
    <line>
      <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<br><Strong><font color=#FF0000>--Failed</font></strong></br>]]>
    </line>
    <line>
      <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<a href='screenshots/Open page Fail.png' target='_new'>Open page Fail.png</a></br>]]>
    </line>
    <line>
      <![CDATA[<br><b>Please look to the screenshot - </b>]]>
    </line>
    <line>
      <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<br><Strong><font color=#FF0000>--Failed</font></strong></br>]]>
    </line>
    <line>
      <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<a href='screenshots/0913195253794Bulkmail_TestCase_01.png' target='_new'>0913195253794Bulkmail_TestCase_01.png</a></br>]]>
    </line>
    <line>
      <![CDATA[<br><b>Please look to the screenshot - </b>]]>
    </line>
    <line>
      <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<br><Strong><font color=#FF0000>--Failed</font></strong></br>]]>
    </line>
    <line>
      <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<a href='screenshots/Open page Fail.png' target='_new'>Open page Fail.png</a></br>]]>
    </line>
    <line>
      <![CDATA[<br><b>Please look to the screenshot - </b>]]>
    </line>
    <line>
      <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<br><Strong><font color=#FF0000>--Failed</font></strong></br>]]>
    </line>
    <line>
      <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<a href='screenshots/0913195314495Bulkmail_TestCase_01.png' target='_new'>0913195314495Bulkmail_TestCase_01.png</a></br>]]>
    </line>
    <line>
      <![CDATA[<br><b>Please look to the screenshot - </b>]]>
    </line>
    <line>
      <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<br><Strong><font color=#FF0000>--Failed</font></strong></br>]]>
    </line>
    <line>
      <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<a href='screenshots/Open page Fail.png' target='_new'>Open page Fail.png</a></br>]]>
    </line>
  </reporter-output>
  <suite name="BSB_Functional_TestCases_Suite" duration-ms="63393" started-at="2020-09-13T14:22:34Z" finished-at="2020-09-13T14:23:37Z">
    <groups>
      <group name="Bulkmail">
        <method signature="Steps.Bulkmail_TestCase_01()[pri:0, instance:com.automation.index.Steps@4d95d2a2]" name="Bulkmail_TestCase_01" class="com.automation.index.Steps"/>
      </group> <!-- Bulkmail -->
    </groups>
    <test name="BSB_Login_Module_TestCases" duration-ms="63393" started-at="2020-09-13T14:22:34Z" finished-at="2020-09-13T14:23:37Z">
      <class name="com.automation.index.Steps">
        <test-method status="PASS" signature="fetchSuiteConfiguration(org.testng.ITestContext)[pri:0, instance:com.automation.index.Steps@4d95d2a2]" name="fetchSuiteConfiguration" is-config="true" duration-ms="11" started-at="2020-09-13T14:22:34Z" finished-at="2020-09-13T14:22:34Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4d3167f4]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fetchSuiteConfiguration -->
        <test-method status="PASS" signature="fetchSuiteConfiguration(org.testng.ITestContext)[pri:0, instance:com.automation.index.Steps@4d95d2a2]" name="fetchSuiteConfiguration" is-config="true" duration-ms="0" started-at="2020-09-13T14:22:34Z" finished-at="2020-09-13T14:22:34Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4d3167f4]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fetchSuiteConfiguration -->
        <test-method status="PASS" signature="setUp(java.lang.reflect.Method, org.testng.ITestContext)[pri:0, instance:com.automation.index.Steps@4d95d2a2]" name="setUp" is-config="true" duration-ms="11418" started-at="2020-09-13T14:22:34Z" finished-at="2020-09-13T14:22:45Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.automation.index.Steps.Bulkmail_TestCase_01() throws java.awt.AWTException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@4d3167f4]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="Bulkmail_TestCase_01()[pri:0, instance:com.automation.index.Steps@4d95d2a2]" name="Bulkmail_TestCase_01" duration-ms="7655" started-at="2020-09-13T14:22:45Z" finished-at="2020-09-13T14:22:53Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.automation.verification.Verify.homepageverify(Verify.java:51)
at com.automation.index.Steps.Bulkmail_TestCase_01(Steps.java:264)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<br><b>Please look to the screenshot - </b>]]>
            </line>
            <line>
              <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<br><Strong><font color=#FF0000>--Failed</font></strong></br>]]>
            </line>
            <line>
              <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<a href='screenshots/Open page Fail.png' target='_new'>Open page Fail.png</a></br>]]>
            </line>
            <line>
              <![CDATA[<br><b>Please look to the screenshot - </b>]]>
            </line>
            <line>
              <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<br><Strong><font color=#FF0000>--Failed</font></strong></br>]]>
            </line>
            <line>
              <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<a href='screenshots/0913195253794Bulkmail_TestCase_01.png' target='_new'>0913195253794Bulkmail_TestCase_01.png</a></br>]]>
            </line>
          </reporter-output>
        </test-method> <!-- Bulkmail_TestCase_01 -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.automation.index.Steps@4d95d2a2]" name="tearDown" is-config="true" duration-ms="5305" started-at="2020-09-13T14:22:53Z" finished-at="2020-09-13T14:22:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Bulkmail_TestCase_01 status=SKIP method=Steps.Bulkmail_TestCase_01()[pri:0, instance:com.automation.index.Steps@4d95d2a2] output=<br><b>Please look to the screenshot - </b>]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp(java.lang.reflect.Method, org.testng.ITestContext)[pri:0, instance:com.automation.index.Steps@4d95d2a2]" name="setUp" is-config="true" duration-ms="9587" started-at="2020-09-13T14:22:59Z" finished-at="2020-09-13T14:23:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.automation.index.Steps.Bulkmail_TestCase_01() throws java.awt.AWTException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@4d3167f4]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="Bulkmail_TestCase_01()[pri:0, instance:com.automation.index.Steps@4d95d2a2]" name="Bulkmail_TestCase_01" duration-ms="5462" started-at="2020-09-13T14:23:08Z" finished-at="2020-09-13T14:23:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.automation.verification.Verify.homepageverify(Verify.java:51)
at com.automation.index.Steps.Bulkmail_TestCase_01(Steps.java:264)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<br><b>Please look to the screenshot - </b>]]>
            </line>
            <line>
              <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<br><Strong><font color=#FF0000>--Failed</font></strong></br>]]>
            </line>
            <line>
              <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<a href='screenshots/Open page Fail.png' target='_new'>Open page Fail.png</a></br>]]>
            </line>
            <line>
              <![CDATA[<br><b>Please look to the screenshot - </b>]]>
            </line>
            <line>
              <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<br><Strong><font color=#FF0000>--Failed</font></strong></br>]]>
            </line>
            <line>
              <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<a href='screenshots/0913195314495Bulkmail_TestCase_01.png' target='_new'>0913195314495Bulkmail_TestCase_01.png</a></br>]]>
            </line>
          </reporter-output>
        </test-method> <!-- Bulkmail_TestCase_01 -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.automation.index.Steps@4d95d2a2]" name="tearDown" is-config="true" duration-ms="5257" started-at="2020-09-13T14:23:14Z" finished-at="2020-09-13T14:23:19Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Bulkmail_TestCase_01 status=SKIP method=Steps.Bulkmail_TestCase_01()[pri:0, instance:com.automation.index.Steps@4d95d2a2] output=<br><b>Please look to the screenshot - </b>]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp(java.lang.reflect.Method, org.testng.ITestContext)[pri:0, instance:com.automation.index.Steps@4d95d2a2]" name="setUp" is-config="true" duration-ms="9531" started-at="2020-09-13T14:23:19Z" finished-at="2020-09-13T14:23:29Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.automation.index.Steps.Bulkmail_TestCase_01() throws java.awt.AWTException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@4d3167f4]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="Bulkmail_TestCase_01()[pri:0, instance:com.automation.index.Steps@4d95d2a2]" name="Bulkmail_TestCase_01" duration-ms="7488" started-at="2020-09-13T14:23:29Z" finished-at="2020-09-13T14:23:36Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.automation.verification.Verify.homepageverify(Verify.java:51)
at com.automation.index.Steps.Bulkmail_TestCase_01(Steps.java:264)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<br><b>Please look to the screenshot - </b>]]>
            </line>
            <line>
              <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<br><Strong><font color=#FF0000>--Failed</font></strong></br>]]>
            </line>
            <line>
              <![CDATA[</br>&nbsp;&nbsp;&nbsp;&nbsp;<a href='screenshots/Open page Fail.png' target='_new'>Open page Fail.png</a></br>]]>
            </line>
          </reporter-output>
        </test-method> <!-- Bulkmail_TestCase_01 -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.automation.index.Steps@4d95d2a2]" name="tearDown" is-config="true" duration-ms="20" started-at="2020-09-13T14:23:37Z" finished-at="2020-09-13T14:23:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Bulkmail_TestCase_01 status=FAILURE method=Steps.Bulkmail_TestCase_01()[pri:0, instance:com.automation.index.Steps@4d95d2a2] output=<br><b>Please look to the screenshot - </b>]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="zipfileupdate()[pri:0, instance:com.automation.index.Steps@4d95d2a2]" name="zipfileupdate" is-config="true" duration-ms="46" started-at="2020-09-13T14:23:37Z" finished-at="2020-09-13T14:23:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- zipfileupdate -->
      </class> <!-- com.automation.index.Steps -->
    </test> <!-- BSB_Login_Module_TestCases -->
  </suite> <!-- BSB_Functional_TestCases_Suite -->
</testng-results>
